helm/alfresco-content-services/charts/alfresco-sync-service/values.yaml

```diff
@@ -88,25 +88,31 @@ database:
   # password:     #ex: alfresco
   # url:    # ex: jdbc:postgresql://oldfashioned-mule-postgresql-acs:5432/alfresco
 
-postgresql:
+postgresql-syncservice:
   ## If true, install the postgresql chart alongside Alfresco Sync service.
   # Note: Set this to false if you use an external database.
   name: postgresql-syncservice
   enabled: true
+  replicaCount: 1
   nameOverride: postgresql-syncservice
-  imageTag: "10.1"
+  imageTag: "11.4.0"
```

Comment:
There's a task to move to Postgres 11.7 (https://issues.alfresco.com/jira/browse/ACS-386) can we please make sure all pods are using 11.7.

Reply:
ok.